### PR TITLE
Properly configure source sets for gradle and fix instrumented test location

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
uia2 integration tests are now broken because of the recent change: https://travis-ci.org/appium/appium-uiautomator2-driver/builds/374725894

Please merge and publish this PR ASAP

@vmaxim @dpgraham FYI

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -61,6 +61,13 @@ android {
         exclude 'META-INF/maven/com.google.guava/guava/pom.properties'
         exclude 'META-INF/maven/com.google.guava/guava/pom.xml'
     }
+    sourceSets {
+        androidTest {
+            java.srcDirs = ['src/androidTestE2eTest/java',
+                            'src/androidTestServer/java',
+                            'src/test/java']
```
Comment (vmaxim) [inline app/build.gradle]: It's unit tests - not Instrumented. They should not be in androidTest section.

Comment (mykola-mokhnach) [reply app/build.gradle]: removed unit tests

File: app/build.gradle [hunk 2]
```diff
@@ -61,6 +61,12 @@ android {
         exclude 'META-INF/maven/com.google.guava/guava/pom.properties'
         exclude 'META-INF/maven/com.google.guava/guava/pom.xml'
     }
+    sourceSets {
+        androidTest {
+            java.srcDirs = ['src/androidTestE2eTest/java',
+                            'src/androidTestServer/java']
```
Comment (vmaxim) [inline app/build.gradle]: ```AppiumUiAutomator2Server``` - it's a ```while(true){}``` test. I think it's a not good idea to include this test in common set.

Comment (mykola-mokhnach) [reply app/build.gradle]: do you know another way to show it in the sources list?

Comment (vmaxim) [reply app/build.gradle]: What do you mean by "sources list"?
There are two build variants in the project. First is the server, second is e2e tests for the server.

Comment (mykola-mokhnach) [reply app/build.gradle]: I mean this file is not shown by Android Studio if we use the default Gradle settings

Comment (mykola-mokhnach) [reply app/build.gradle]: scr

Comment (vmaxim) [reply app/build.gradle]: image

Comment (mykola-mokhnach) [reply app/build.gradle]: it is visible in the project tree, but is not included into Android sources. This is how it looks without that Gradle entry:

scr2

Comment (mykola-mokhnach) [reply app/build.gradle]: I';ve moved the class to main source set if androidTests is not the best place for it

Comment (vmaxim) [reply app/build.gradle]: image

**Pull request merged**